iftk/helpers/deepgram_stream_queue.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import asyncio
+import json
+import pyaudio
+import websockets
+
+FORMAT = pyaudio.paInt16
+CHANNELS = 1
+RATE = 44100
+CHUNK = 512
+DEEPGRAM_CHUNK = 8000
+DEEPGRAM_RATE = 16000
+SECONDS = 2
+WSS_URL = "wss://api.deepgram.com/v1/listen?endpointing=500&encoding=linear16&sample_rate=16000&channels=1&interim_results=false"
+
+
+async def DeepgramStream(key: str, transcription_queue: asyncio.Queue) -> None:
```

Comment:
naming nit: functions should be snake_case
```suggestion
async def deepgram_stream(key: str, transcription_queue: asyncio.Queue) -> None:
```

Also, this function should probably take an AsyncIterator of audio events and return an AsyncIterator of transcriptions. It's OK to do that in a future PR.